src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/MapsPage.java

```diff
@@ -0,0 +1,53 @@
+package com.qaprosoft.carina.demo.mobile.gui.pages.ios;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.foundation.webdriver.locator.ExtendedFindBy;
+import com.qaprosoft.carina.demo.mobile.gui.pages.common.MapsPageBase;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebDriver;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = MapsPageBase.class)
+public class MapsPage  extends MapsPageBase {
```

Comment:
remove redundant space between MapsPage and extends

Reply:
done